src/api-client.js

```diff
@@ -10,28 +10,28 @@ export default {
   },
 
   uploadSubmissions(labId, formData) {
-    return axios.post(`http://localhost:5000/results/cc451/${labId}`, formData);
+    return axios.post(`${graderx_api}results/cc451/${labId}`, formData);
```

Comment:
I think it should be `${GRADERX_API}/` note the trailing slash. just to make it very clear it's a base URL